providers/org/zenodo/harvester.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from furl import furl
+
+from share.harvest.oai import OAIHarvester
+
+logger = logging.getLogger(__name__)
+
+
+class ZenodoHarvester(OAIHarvester):
```

Comment:
Maybe an init with the args and redefining them? Then you'd only have to rewrite the init in each class instead of do_harvest as well?